compose.override.dist.yml

```diff
@@ -0,0 +1,56 @@
+version: "3.8"
+services:
+  php:
+    image: ghcr.io/sylius/sylius-php:8.2-fixuid-alpine
+    user: ${DOCKER_USER:-1000:1000}
+    depends_on:
+      mysql:
+        condition: service_healthy
+    environment:
+      APP_DEBUG: 0
+      APP_ENV: ${ENV:-prod}
+      APP_SECRET: EDITME
+      DATABASE_URL: "mysql://root@mysql/sylius_%kernel.environment%"
+      MAILER_URL: null://null
+      MESSENGER_TRANSPORT_DSN: doctrine://default
+      SYLIUS_MESSENGER_TRANSPORT_MAIN_DSN: doctrine://default
+      SYLIUS_MESSENGER_TRANSPORT_MAIN_FAILED_DSN: doctrine://default?queue_name=main_failed
+      SYLIUS_MESSENGER_TRANSPORT_CATALOG_PROMOTION_REMOVAL_DSN: doctrine://default?queue_name=catalog_promotion_removal
+      SYLIUS_MESSENGER_TRANSPORT_CATALOG_PROMOTION_REMOVAL_FAILED_DSN: doctrine://default?queue_name=catalog_promotion_removal_failed
+      PHP_DATE_TIMEZONE: ${PHP_DATE_TIMEZONE:-UTC}
```

Comment:
```
        env_file:
            - .env.local
```

We can create an example `.env.local` and add it to the gitignore, as we de with `.override.dist.yml`

Reply:
I believe it's better to keep all environment variables in one place, without separating them intoanother file or using an example file. I have added a comment suggesting that if somebody wishes, they can move these environment variables to `.env.local`.